frappe/utils/fixtures.py

```diff
@@ -67,20 +67,34 @@ def export_fixtures(app=None):
 	else:
 		apps = frappe.get_installed_apps()
 	for app in apps:
-		for fixture in frappe.get_hooks("fixtures", app_name=app):
+		fixture_auto_order = bool(frappe.get_hooks("fixture_auto_order", app_name=app))
+		fixtures = frappe.get_hooks("fixtures", app_name=app)
+		for index, fixture in enumerate(fixtures):
 			filters = None
 			or_filters = None
 			if isinstance(fixture, dict):
 				filters = fixture.get("filters")
 				or_filters = fixture.get("or_filters")
+				prefix = fixture.get("prefix")
```

Comment:
@Tom-Finke Can you add some kinda document and test on how this is supposed to be used? 

- ref: https://github.com/frappe/frappe/blob/develop/frappe/tests/test_fixture_import.py

Reply:
@ankush  I pushed a new commit, unfortunately I couldn't push it directly into this PR so I created a new one...
See https://github.com/frappe/frappe/pull/22567